docker-assets/container-scripts/restore_db

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+# Defaults
+# Notes : 
+# userdata is a directory created by the PG SCL image when the DB is first initialized on pod
+# latest is a symlink created by the backup jobscript pointing to the latest successful backup directory
+
+BACKUP_VERSION=${BACKUP_VERSION:-latest}
+PG_SVC_NAME=${PG_SVC_NAME:-postgresql}
```

Comment:
@carbonin I would avoid all destructive actions with the existing data. My idea was to incurr the least downtime as possible, tar/compressing the existing data could potentially be a time consuming task, a rename seemed like a quick and safe approach.

In regards of the PG_SVC_NAME , I'm fine either way. Trying to thing benefits from one way or the other...